packages/hawtio/src/plugins/connect/ConnectionStatus.tsx

```diff
@@ -35,13 +36,25 @@ export const ConnectionStatus: React.FunctionComponent = () => {
   let icon = null
   switch (reachable) {
     case 'reachable':
-      icon = <PluggedIcon color='green' />
+      icon = (
+        <Icon style={{ color: 'var(--pf-v5-global--success-color--100)' }}>
+          <PluggedIcon />
+        </Icon>
+      )
       break
     case 'not-reachable':
-      icon = <UnpluggedIcon color='red' />
+      icon = (
+        <Icon style={{ color: 'var(--pf-v5-global--danger-color--100)' }}>
```

Comment:
why can't we use `color={'xxx'}` here?

Reply:
that's my change. I didn't find proper color and I was using it from https://www.patternfly.org/components/alert/#prefixed-with-pf-v5-c-alert

Reply:
I updated the rest icons with same as Grzegorz because I had all of them black.